cmd/wio/commands/create/helper.go

```diff
@@ -38,12 +39,20 @@ func getBoard(boardProvided string) string {
 
 func (info createInfo) fillReadMe(queue *log.Queue, readmeFile string) error {
     log.Verb(queue, "filling README file ... ")
+
+    platform := info.platform
+    if info.projectType == constants.APP {
+        if platform == "all" {
+            platform = getPlatform(platform)
+        } else {
+            platform += " "
```

Comment:
why add a space? and why is this logic only needed for Apps?

Reply:
we do not want a text saying "wio all application" so I made it such that if it is all it will just become "wio application". For pkg projects, this string is not used at all